lib/widgets/nutrition/charts.dart

```diff
@@ -19,12 +19,128 @@
 import 'package:fl_chart/fl_chart.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
-import 'package:intl/intl.dart';
 import 'package:wger/helpers/colors.dart';
 import 'package:wger/models/nutrition/nutritional_plan.dart';
 import 'package:wger/models/nutrition/nutritional_values.dart';
 import 'package:wger/widgets/measurements/charts.dart';
 
+class FlNutritionalPlanGoalWidget extends StatefulWidget {
+  const FlNutritionalPlanGoalWidget({
+    super.key,
+    required NutritionalPlan nutritionalPlan,
+  }) : _nutritionalPlan = nutritionalPlan;
+
+  final NutritionalPlan _nutritionalPlan;
+
+  @override
+  State<StatefulWidget> createState() => FlNutritionalPlanGoalWidgetState();
+}
+
+// * fl_chart doesn't support horizontal bar charts yet.
```

Comment:
I think I mentioned it on discord before, but if we have to contort ourselves for such things, we could just use https://pub.dev/packages/graphic for such things, which seems to support this natively (same with time series, which cause a bit of pain with fl)

Reply:
I'll have a look at graphic. Though I'm quite pleased with how simple and elegant the "diy gauge" approach is.. I doubt it'll be as straightforward with the graphic package

Reply:
sure, this was just a suggestion in case we need something else in the future